BASE/src/Microsoft.ApplicationInsights/DataContracts/ExceptionTelemetry.cs

```diff
@@ -16,7 +17,7 @@
     /// <remarks>
     /// Additional exception details will need to be tracked manually.
     /// </remarks>
-    public sealed class ExceptionTelemetry : ITelemetry, ISupportProperties, ISupportAdvancedSampling, ISupportMetrics, IAiSerializableTelemetry
+    public sealed class ExceptionTelemetry : OperationTelemetry, ITelemetry, ISupportProperties, ISupportAdvancedSampling, ISupportMetrics, IAiSerializableTelemetry
```

Comment:
I think`ExceptionTelemetry` is not supposed to be an `OperationTelemetry` (as it is not a duration)?

Reply:
You're correct. I inherited it and kept all its value as default, considering our backend may look out of these values. Validated with test app, we don't need these attributes. Removing OperationTelemetry reference on both ExceptionTelemetry and TraceTelemetry.

Reply:
Right. Exception, Trace don't represent an operation, so they dont need to inherit operationtelemetry.

Reply:
Fixed